### PR TITLE
Auto OCP-29477

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei  could you help to review?  Thanks

### CODE REVIEW

File: lib/rules/web/admin_console/4.5/operator_hub.xyaml [hunk 1]
```diff
@@ -266,3 +266,9 @@ open_operator_modal:
     keyword: <operator_name>
   action: filter_by_keyword_on_catalog_page
   action: open_first_card_in_overlay
+click_checkbox_from_providertype:
+  params:
+    filter_type_value: providerType
+  action: wait_filter_section_loaded
+  action: choose_one_checkbox
+
```
Comment (yanpzhan) [inline lib/rules/web/admin_console/4.5/operator_hub.xyaml:274]: There is an existing rule: click_checkbox_from_provider_type, could you check if it's the same function with "click_checkbox_from_providertype"

Comment (liangxia) [reply lib/rules/web/admin_console/4.5/operator_hub.xyaml:282]: What's the change for ?

**Pull request merged**